src/libsystemd/sd-network/sd-network.c

```diff
@@ -204,6 +204,25 @@ _public_ int sd_network_link_get_route_domains(int ifindex, char ***ret) {
         return network_link_get_strv(ifindex, "ROUTE_DOMAINS", ret);
 }
 
+_public_ int sd_network_link_get_dns_default_route(int ifindex) {
```

Comment:
get → is ? "get" sounds too much like something more than yes/no would be returned.

Reply:
So I thought about this, but we have sd_network_link_get_required_for_online() already which also returns a boolean, so I stuck to the same naming logic there.

THis stuff isn't exported, I figured, and it wasn't clear to me that "is" is really that better here, as after all the link "is" not necessarily the DNS default route, but it is just supposed to be, from networkd's pov...

Anyway, so I was in doubt, and it's private, hence I figured we should stick to the current naming scheme. If we make sd-network public we can of course revisit this...

Reply:
Ack.